core/src/main/scala/backbone/javadsl/ConsumerSettings.scala

```diff
@@ -0,0 +1,21 @@
+package backbone.javadsl
+
+import java.util.{List => JList, Optional => JOption}
+
+import backbone.consumer.Limitation
+
+/**
+ *
+ * @param events        a java list of events to listen to
+ * @param topics        a java list of topics to subscribe to
+ * @param queue         the name of a queue to consume from
+ * @param parallelism   number of concurrent messages in process
+ * @param consumeWithin optional limitation when backbone should stop consuming
+ */
+case class ConsumerSettings(
```

Comment:
Maybe we don't have to provide a separate case class but simply provide a .create method for the scala one (moving it to the base package)